macros/helpers/filament_swap.cfg

```diff
@@ -159,12 +167,14 @@ gcode:
         {% endif %}
     {% endif %}
 
-    {% if klippain_ercf_enabled %}
-        {% if printer.ercf.enabled %}
-            {% if printer.ercf.clog_detection > 0 %}
-                ERCF_TEST_CONFIG enable_clog_detection=0
-                {% if verbose %}
-                    RESPOND MSG="ERCF clog detection deactivated for filament tip shaping"
+    {% if klippain_mmu_enabled %}
+        {% if printer.mmu.enabled %}
+            {% if printer.mmu.sync_drive is false %}
+                {% if printer.mmu.clog_detection > 0 %}
+                    MMU_TEST_CONFIG enable_clog_detection=0
+                    {% if verbose %}
+                        RESPOND MSG="MMU clog detection deactivated for filament tip shaping"
+                    {% endif %}
                 {% endif %}
             {% endif %}
         {% endif %}
```

Comment:
Same here: do you think it's possible to merge this into (and use) the `_KLIPPAIN_MMU_SET_CLOGDETECTION` macro that is very similar and already used for the purge line?